### PR TITLE
Add CKAN cronjobs

### PR DESCRIPTION
**Pull request by boffbowsh**
Includes a helper type for defining jobs that are calls to paster commands.

@andrewgarner @brucebolt is this easy enough to add new jobs as needed?

### CODE REVIEW

File: modules/govuk/manifests/apps/ckan/cronjobs.pp [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# == Class: govuk::apps::ckan::cronjobs
+#
+class govuk::apps::ckan::cronjobs {
+
+  govuk::apps::ckan::paster_cronjob { 'harvester':
```
Comment (andrewgarner) [inline modules/govuk/manifests/apps/ckan/cronjobs.pp]: Do we need the `harvester clean_harvest_log` task too? It's documented – https://github.com/ckan/ckanext-harvest#setting-up-the-harvesters-on-a-production-server

**Comment by boffbowsh** [comment]
@andrewgarner it doesn't run in prod at the moment, but I've added it.

File: modules/govuk/manifests/apps/ckan/cronjobs.pp [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+class govuk::apps::ckan::cronjobs {
+
+  govuk::apps::ckan::paster_cronjob { 'harvester run':
+    paster_command => 'harvester run',
+    plugin         => 'ckanext-harvest',
+    minute         => '*/10',
+  }
+
+  govuk::apps::ckan::paster_cronjob { 'harvester cleanup':
+    paster_command => 'clean_harvest_log',
```
Comment (andrewgarner) [inline modules/govuk/manifests/apps/ckan/cronjobs.pp]: `harvester clean_harvest_log`

File: modules/govuk/manifests/apps/ckan/cronjobs.pp [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+  govuk::apps::ckan::paster_cronjob { 'harvester cleanup':
+    paster_command => 'harvester clean_harvest_log',
+    plugin         => 'ckanext-harvest',
+    hour           => '2',
+  }
+
+  govuk::apps::ckan::paster_cronjob { 'archiver':
+    paster_command => 'archiver update',
+    plugin         => 'ckanext-archiver',
```
Comment (andrewgarner) [inline modules/govuk/manifests/apps/ckan/cronjobs.pp]: I don't think we currently have the `ckanext-archiver` available.

Comment (andrewgarner) [reply modules/govuk/manifests/apps/ckan/cronjobs.pp]: I think we can leave this one out.

**Comment by boffbowsh** [comment]
Updated.

**Comment by boffbowsh** [comment]
@brucebolt can you review so I can merge this? It's my only open alphagov PR at the moment

**Pull request merged**